### PR TITLE
Added experiment config for sept_release branch

### PR DESCRIPTION
**Pull request by RuchikaSharma-8**
### Description
Modified the name, branch and flags of an existing experiment configuration to run experiments on sept_release branch.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - Ran the build script from the "periodic_experiments" folder using command "./build.sh" after setting EXPERIMENT_NUMBER to "2". Ran all list tests and 1 load test.  Ensured that the tests were running as expected on newly added configuration.
2. Unit tests - NA
3. Integration tests - NA

### CODE REVIEW

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json [hunk 1]
```diff
@@ -7,9 +7,9 @@
       "end_date": "2023-12-30 05:30:00+00:00"
     },
     {
-      "config_name": "DisabledStorageClientLibrary",
-      "gcsfuse_flags": "--implicit-dirs --enable-storage-client-library=false --debug_fuse --debug_gcs",
-      "branch": "master",
+      "config_name": "SeptReleaseEnabledStorageClientLibrary",
```
Comment (sethiay) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json:10]: let's name it SeptRelease and remove  '--enable-storage-client-library' flag from it

File: perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json [hunk 2]
```diff
@@ -7,9 +7,9 @@
...
+      "gcsfuse_flags": "--implicit-dirs --enable-storage-client-library --debug_fuse --debug_gcs",
+      "branch": "sept_release",
       "end_date": "2023-12-30 05:30:00+00:00"
```
Comment (sethiay) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json:13]: As the name suggests, it is meant for September release, so end_date should be 2023-09-30. Also, remove time from the end_date.

Comment (raj-prince) [inline perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json:13]: Why is end_date same for all configs? What does it signify?

Comment (raj-prince) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json:13]: We can change the date to September or October first week.

Comment (raj-prince) [reply perfmetrics/scripts/continuous_test/gcp_ubuntu/periodic_experiments/experiments_configuration.json:13]: +1

**Review by raj-prince — APPROVED** [review]
Please incorporate the Ayush's comment.

**Review by raj-prince — APPROVED** [review]

**Pull request merged**